### PR TITLE
Flesh out documentation

### PR DESCRIPTION
**Pull request by alfonsomunozpomer**
I pulled the cell clusters sections before the marker genes since the former is required to run the latter. I documented the `CONDENSED_SDRF_TSV` variable which gave me some headaches. I fixed some minor typos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,140 +1,140 @@
...
 
 For direct usage, this module requires Rscript (with optparse and tidyr, in Ubuntu and Debian-based distributions install packages `r-cran-optparse` and `r-cran-tidyr`), psql and node.
 
 # `scxa_analytics` Table
 
 ## Create schema
 
-Currently, schema for postgres scxa-* tables is stored at https://github.com/ebi-gene-expression-group/atlas/tree/dev/atlas-misc/scripts/db_updates. In time, the schema will be either consumed
-from this or a dedicated repo.
+Currently, schema for postgres scxa-* tables is stored at https://github.com/ebi-gene-expression-group/atlas/tree/dev/atlas-misc/scripts/db_updates. In time, the schema will be either consumed from this or a dedicated repo.
```
Comment (pinin4fjords) [inline README.md]: Do we need to point this line at atlas-schemas instead now?

Comment (pcm32) [inline README.md]:
```suggestion
Currently, schema for postgres scxa-* tables is stored at https://github.com/ebi-gene-expression-group/atlas-schemas/tree/master/flyway/scxa.
```
Agreed!

**Review by pcm32 — APPROVED** [review]
LGTM, thanks @alfonsomunozpomer!

**Comment by pcm32** [comment]
I hadn't realised that the test were not passing...

**Comment by pcm32** [comment]
dammed... had made the fix on another branch....

**Comment by alfonsomunozpomer** [comment]
@pcm32 @pinin4fjords What’s your policy here? Should I be the one to press the merge button? Do you usually squash PRs?

**Comment by pinin4fjords** [comment]
> @pcm32 @pinin4fjords What’s your policy here? Should I be the one to press the merge button? Do you usually squash PRs?

We leave it to the proposer to merge so they get the commit credit, and yes- squash.

**Pull request merged**